src/test/resources/unit/test_project/pom.xml

```diff
@@ -26,14 +26,20 @@
             <plugin>
                 <groupId>com.exasol</groupId>
                 <artifactId>artifact-reference-checker-maven-plugin</artifactId>
-                <version>0.2.0</version>
+                <version>0.3.0</version>
                 <executions>
                     <execution>
                         <goals>
                             <goal>verify</goal>
                         </goals>
                     </execution>
                 </executions>
+                <configuration>
+                    <excludes>
+                        <exclude>/excluded/*</exclude>
+                        <exclude>excluded_invalid.md</exclude>
```

Comment:
Should not you also put nested here, `nested/excluded_invalid.md`?
Or does it match it in the path anyway?

Reply:
since it does not start with a `/` it matches in all subdirectories.

Reply:
I now tried to clarify it above.